egs2/mr_openslr64/asr1/conf/tuning/train_asr_conformer5.yaml

```diff
@@ -0,0 +1,109 @@
+cudnn_benchmark: false 
```

Comment:
Can you simplify the config and combine some of them? We expect the config to be aligned with the results you report